distributed/http/scheduler/prometheus.py

```diff
@@ -68,6 +69,9 @@ def collect(self):
         yield tasks
 
 
+COLLECTORS = [_PrometheusCollector, SemaphoreMetricExtension]
```

Comment:
I like the approach since it's simple and very similar to how the scheduler extensions are defined

Reply:
yeah I got my inspiration from the scheduler extensions